howto/manage-groups.md

```diff
@@ -37,21 +42,21 @@ jimmctl auth relation add group-A#member member group-B
 To add a new group to your JIMM controller, use the `auth group add` command followed by the name you want to assign to the group. For example:
 
 ```text
-jimmctl auth group add A
+juju add-group A
 ```
 
-> See more: {ref}`jimmctl auth group add <summary-5>`
+> See more: {ref}`juju add-group <summary-5>`
 
 (view-all-the-current-groups)=
 ## View all the current groups
 
 To view all the current groups, run the `auth group list` command. For example:
```

Comment:
Here and in other places: The text introducing the code doesn't seem to have been updated.

```suggestion
To view all the current groups, run the `list-groups` command. For example:
```